measures/2018/measures-data.json

```diff
@@ -7630,7 +8178,9 @@
     "lastPerformanceYear": null,
     "isHighPriority": false,
     "isInverse": false,
-    "overallAlgorithm": "overallStratumOnly",
+    "overallAlgorithm": "simpleAverage",
```

Comment:
I don't think you meant to change this.

Reply:
Per Sarah's comment: https://jira.cms.gov/browse/QPPSF-2594?focusedCommentId=873248&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-873248 `overallAlgorithm` was changed in the source file.